css/404.css

```
@@ -1,15 +1,15 @@
 body {
-    background-color: #23272A;
+    background-color: #1B2430;
```

```suggestion
    background-color: #5c18ae;
``` 
Maybe try this purple, the website looks fine from the preview, but just very dark.